.github/workflows/documentation.yaml

```diff
@@ -13,9 +13,10 @@ jobs:
           pip install sphinx
           python -m pip install sphinx-autoapi
           python -m pip install sphinx_rtd_theme
+          python -m pip install tox tox-gh-actions
       - name: Sphinx build
         run: |
-          sphinx-build doc _build
+          tox -e docs
       - name: Deploy
         uses: peaceiris/actions-gh-pages@f09b4d7e2f12f433806a6273c6b0025df1f7859d
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
I do not know if the path will change. But if the build failed, I would suggest updating `publish_dir:` to `publish_dir: .tox/docs/tmp/html/` (at least this was tested in different repository.

Reply:
This should be on line 26!